infra/main.bicep

```diff
@@ -0,0 +1,421 @@
+targetScope = 'subscription'
+
+@minLength(1)
+@maxLength(64)
+@description('Name of the the environment which is used to generate a short unique hash used in all resources.')
+param environmentName string
+
+@description('provide a 2-13 character prefix for all resources.')
+param resourcePrefix string = environmentName
```

Comment:
Instead of using resourcePrifix like this, use the method of generating the prefix like we do in azd templates.